doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -1586,3 +1586,104 @@ def write_cdb(port,cmd):
     write_reg(port, LPLPAGE, INIT_OFFSET+CMDLEN, len(cmd)-CMDLEN, cmd[CMDLEN:])
     write_reg(port, LPLPAGE, INIT_OFFSET, CMDLEN, cmd[:CMDLEN])
 ```
+
+### 7.Performance Monitoring in 400-G ZR module
+#### 7.1 Overview
+Performance monitoring in 400G ZR/CCMIS optical modules is essential for detecting link degradation and correction. It can be used to compare optical link performance against desired parameters and benchmarks, providing valuable insight into the overall health of the interface link. Below sub-sections will walk through the CLI syntax, output format and high level design.
+
+#### 7.2 PM parameters
+Please refer to the [2.1.5 Transceiver PM Table](https://github.com/sonic-net/SONiC/edit/master/doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md#215-transceiver-pm-table) for the parameters that will be monitored with this CLI.
+
+ 
+#### 7.3 CLI Sub-options and Syntax 
+```
+#show int transceiver <enter>​
+commands:​
+pm show interface transceiver performance monitoring​
+​
+#Show int trans pm <enter>​
+commands:​
+Current show current pm data​
+history show historical pm data​
+
+​
+#show int trans pm current <predefined window period>​
+commands:​
+60sec show cumulative pm statistics for 60sec time window for the current window. ​
+15min show cumulative pm statistics for 15min time window for the current window.​
+24Hr  show cumulative pm statistics for 24Hr time window for the current window.​
+<end> Without time window, the CLI will display the current snapshot of pm parameter.​
+
+#show int trans pm history​
+commands:​
+60sec show cumulative pm statistics for 60sec time window for the given window number.​
+15min show cumulative pm statistics for 15min time window for the given window number.​
+24Hr  show cumulative pm statistics for 24Hr time window for the given window number.​
+​
+#show int trans pm history 30sec window <predefined window period number> -n asic0 Ethernet0​
```

Comment:
can we add something to describe the mapping between asic number and port number?